modules/deploy/pages/deployment-option/cloud/monitor-cloud.adoc

```diff
@@ -0,0 +1,81 @@
+
+You can monitor Redpanda with Prometheus or with any other monitoring and alerting tool, such as Datadog, New Relic, Elastic Cloud, Google, or Azure.
+
+[NOTE]
+====
+To maximize performance, Redpanda exports some metrics only when the underlying feature is in use. For example, a metric for consumer groups, xref:reference:public-metrics-reference.adoc#redpanda_kafka_consumer_group_committed_offset[`redpanda_kafka_consumer_group_committed_offset`], is only exported when groups are registered.
+====
+
+== Configure Prometheus
+
+To monitor a Dedicated or BYOC cluster in https://prometheus.io/[Prometheus^]:
+
+. On the Redpanda Cloud *Overview* page for your cluster, under *How to connect*, click the *Prometheus* tab. 
+
+. Click the copy icon for *Prometheus YAML* to copy the contents to your clipboard. 
++
+image::shared:cloud_metrics.png[How to connect screenshot]
++
+The YAML contains the Prometheus scrape target configuration, as well as authentication, for the cluster.
++
+[,yaml]
+----
+- job_name: redpandaCloud-sample
+  static_configs:
+    - targets:
+        - console-<id>.<identifier>.fmc.cloud.redpanda.com
+  metrics_path: /api/cloud/prometheus/public_metrics
+  basic_auth:
+    username: prometheus
+    password: "<prom_pass>"
+  scheme: https
+----
+
+. Save this information to Prometheus, and observe in Prometheus that metrics from Redpanda endpoints are scraped.
+
+== Configure Datadog
+
+To monitor a Dedicated or BYOC cluster in https://www.datadoghq.com/[Datadog]:
+
+. On the Redpanda Cloud *Overview* page for your cluster, under *How to connect*, click the *Prometheus* tab. 
+
+. Click the copy icon for *Prometheus YAML* to find scrape target configuration, as well as authentication, for the cluster.
+
+. In Datadog, define the `openmetrics_endpoint` URL for that monitored cluster. The integration configuration should look similar to the following:
++
+[,yaml]
+----
+instances:
+  # The endpoint to collect metrics about Polaris. Replace ORGANIZATION_NAME with the name of your organization.
+  - openmetrics_endpoint: https://console-<id>.<identifier>.fmc.cloud.redpanda.com/api/cloud/prometheus/public_metrics
+    use_openmetrics: true
+    collect_counters_with_distributions: true
+
+    auth_type: basic
+    username: prom_user
+    password: prom_pass
+----
+
+. Restart the Datadog agent. 
+
+NOTE: Because the Prometheus endpoint in Redpanda Cloud provides a scrape that includes all brokers, one Datadog agent must be running to monitor it. The agent must run in a container in your own container infrastructure. Redpanda does not support launching this container inside a Dedicated or BYOC Kubernetes cluster. For more information, see the https://docs.datadoghq.com/integrations/redpanda/?tab=host[Datadog documentation^] and https://github.com/DataDog/integrations-extras/tree/master/redpanda[Redpanda Datadog integration^].
```

Comment:
> Because the Prometheus endpoint in Redpanda Cloud provides a scrape that includes all brokers...

"provides a scrape" sounds strange to me. Is "scrape" a DataDog thing or is this a wordy way of saying something like "the Prometheus endpoint in Redpanda Cloud aggregates metrics from all brokers"?

Reply:
I think this is a familiar term with Prometheus. See [here](https://prometheus.io/docs/prometheus/latest/getting_started/).

Reply:
@voutilad we could remove from this Datadog section. How's this? 

Because the Prometheus endpoint in Redpanda Cloud includes all brokers...

Reply:
I think it's just a different way to say "obtain data" . Scrape is definetely weird because it can sound like you have to do some action to retrieve the data. But if it's a Prometheus lingo, I would keep it.

Reply:
See https://docs.datadoghq.com/containers/kubernetes/prometheus/?tab=kubernetesadv2
Looks like Prometheus lingo.